src/Umbraco.Web.UI/Umbraco/Views/Default.cshtml

```diff
@@ -17,11 +16,12 @@
             isDebug = true;
         }
     }
+    var runtimeMinifier = Current.RuntimeMinifier;
 
-    Html
-        .RequiresCss("assets/css/umbraco.css", "Umbraco")
-        .RequiresCss("lib/bootstrap-social/bootstrap-social.css", "Umbraco")
-        .RequiresCss("lib/font-awesome/css/font-awesome.min.css", "Umbraco");
+    runtimeMinifier.RequiresCss("umbraco",
```

Comment:
This needs to be a pre-defined bundle... as in, defined in c# on startup. View based declarations will not work in load balancing (see prev comments). I'll update.